### DISCUSSION TITLE
Can the <q-field> component be made into this style?

### DISCUSSION THREAD

**Discussion by jinmarcus** [quasarframework/quasar#15230] [Ideas / Proposals]
Can the `<q-field>` component be made into this style?

Put the label tag at the top of the input box

This can make the `<q-field>` component appear less compact

I think it is possible to modify the native `<fieldset>` tag to achieve

```
<fieldset>
  <legend>Email</legend>
</fieldset>
```

like this

![image](https://user-images.githubusercontent.com/17996289/211121590-7b20b56e-0a5b-4ada-80f0-4e091122f973.png)

**Comment by metalsadman** [comment]
Yeah using some css, adjusting the translateY percentage.
https://codepen.io/metalsadman/pen/MWBJgBE

**Reply by jinmarcus** [reply]
@metalsadman
that's what i do now
but there is a problem
The label of the `<q-field>` does not change after the background color is changed
always white

**Reply by metalsadman** [reply]
you have to somehow get either the parent background-color or the body.
